fabric/control-api/json/services/listener.json

```
@@ -20,10 +20,7 @@
         "X25519:P-256:P-521:P-384"
     ],
     "subject_names": [
-      "spiffe://{{ .Values.global.spire.trust_domain}}/edge"{{- if and (eq .service.serviceName "jwt-security") (.Values.global.data.external.enabled) }},
-      "spiffe://{{ .Values.global.spire.trust_domain}}/data"{{- else if eq .service.serviceName "data-internal"}},
-      "spiffe://{{ .Values.global.spire.trust_domain}}/catalog"{{- else if and (eq .service.serviceName "internal-jwt-security") (.Values.global.data.internal.enabled) }},
-      "spiffe://{{ .Values.global.spire.trust_domain}}/data-internal"{{- end }}
+      "spiffe://{{ .Values.global.spire.trust_domain}}/edge"
```

this will actually still need a check for jwt so data can connect to jwt

```suggestion
      "spiffe://{{ .Values.global.spire.trust_domain}}/edge"{{- if and (eq .service.serviceName "jwt-security") (.Values.global.data.external.enabled) }},
      "spiffe://{{ .Values.global.spire.trust_domain}}/data"
```

---

Im not sure how to handle the case that "external" data is not installed? without the data.external.enabled global option

---

`data` is installed by default now... should I have kept the option to disable it?

---

im not sure - the ability to disable it was an [issue](https://github.com/DecipherNow/helm-charts/issues/646),  [PR](https://github.com/DecipherNow/helm-charts/pull/664). @chrisbsmith @moosecanswim thoughts on whether or not Data should always be installed by default?

---

No, data should not be installed by default.  Now that we're using Redis, no gm-datas should be installed with a default config.

---

Ah okay thanks. I will restore the option to toggle it and add a check for it in this template